client/db/src/light.rs

```diff
@@ -576,6 +576,9 @@ impl<Block> LightBlockchainStorage<Block> for LightStorage<Block>
 				writes: io_stats.writes,
 				reads: io_stats.reads,
 				average_transaction_size: io_stats.avg_transaction_size() as u64,
+				/// Light client does not track those
```

Comment:
```suggestion
				// Light client does not track those
```

(`///` would associate it with just one of "those" two fields)